packages/@jsii/spec/lib/assembly.ts

```diff
@@ -148,7 +148,19 @@ export interface Assembly extends AssemblyConfiguration, Documentable {
 /**
  * Shareable configuration of a jsii Assembly.
  */
-export interface AssemblyConfiguration {
+export interface AssemblyConfiguration extends Targetable {
+  /**
+   * Submodules declared in this assembly.
+   *
+   * @default none
+   */
+  submodules?: { [fqn: string]: SourceLocatable & Targetable };
```

Comment:
Where is the `readme` contents going to be? Is it under `Targetable`?

Reply:
Could be added to `Targetable` (moved there from `Assembly`) yeah...

I would do this when I actually start processing READMEs in a subsequent PR. I don't want to bloat this one :)